src/Compilers/Core/Portable/CommandLine/CommonCommandLineParser.cs

```diff
@@ -2,30 +2,29 @@
 
 using System;
 using System.Collections.Generic;
+using System.Collections.Immutable;
 using System.Diagnostics;
 using System.Globalization;
 using System.IO;
 using System.Linq;
 using System.Text;
-using Microsoft.CodeAnalysis.Collections;
 using Microsoft.CodeAnalysis.PooledObjects;
 using Microsoft.CodeAnalysis.Text;
 using Roslyn.Utilities;
-using System.Collections.Immutable;
 
 namespace Microsoft.CodeAnalysis
 {
     public abstract class CommandLineParser
     {
         private readonly CommonMessageProvider _messageProvider;
-        internal readonly bool IsScriptRunner;
+        internal readonly bool IsScript;
```

Comment:
I thought you're only gonna rename the property. "IsScript" sounds weird in this context. The parser isn't a script.

Reply:
True, though `IsScriptRunner` was already weird by that logic. The parser isn't a script runner. :smile:

I changed it because I wanted to keep the names consistent. Having the command-line parsers pass along `isScriptRunner: true` as an argument for `Script` seems pretty inconsistent to me. After all, this is about parsing a command-line for "script". Also, I think this reads much better now:

```C#
IsScript ? SourceCodeKind.Script : SourceCodeKind.Regular
```

I'm OK with changing it to something else. Do you have any suggestions? Maybe go with something generic like `InScriptParsingMode`?

Reply:
```IsScriptRunnerCommandLineParser``` was a little bit too long :) 
```IsScriptRunnerMode``` ?

Reply:
You really like this "Runner" notion, huh? :smile: I'll comply but I do feel this makes things arbitrarily inconsistent. Is "Runner" used elsewhere in the code base to strengthen that terminology?

I'll make the change tomorrow when I'm back in the office.

@Pilchie, is this approved for ask mode?

Reply:
FWIW, I'm going to go ahead and use `IsScriptCommandLineParser`. 😀

Reply:
Sounds good.